tasks/src/reinforcement-learning/about.md

```diff
@@ -1,6 +1,10 @@
 ## Use Cases
 
-You can contribute this area with common use cases of the task! 
+### Gaming
+
+Reinforcement learning is known for its application to video games. Since the games provide a safe environment to the agent to be trained in the sense that it is perfectly defined and controllable. This makes them perfect candidates for experimentation and will help a lot to learn about the capabilities and limitations of various RL algorithms. 

```

Comment:
```suggestion
Reinforcement learning is known for its application to video games. Since the games provide a safe environment for the agent to be trained in the sense that it is perfectly defined and controllable, this makes them perfect candidates for experimentation and will help a lot to learn about the capabilities and limitations of various RL algorithms. 

```